qiskit_braket_provider/providers/braket_job.py

```diff
@@ -87,7 +89,7 @@ def __init__(
         tasks: Union[List[LocalQuantumTask], List[AwsQuantumTask]],
         **metadata: Optional[dict]
     ):
-        """AWSBraketJob for local execution of circuits.
+        """AmazonBraketTask for local execution of circuits.
 
         Args:
             job_id: id of the job
```

Comment:
Could we rename the arg from job_id to task_id?

Reply:
yes  I will do it now

Reply:
@kshitijc everywhere in the folder?

Reply:
this also should be completed via the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/c0450cc4b752c71a34e5c147c9d720e797ccdb24